src/core/MOM_PressureForce_FV.F90

```diff
@@ -141,12 +145,15 @@ subroutine PressureForce_FV_nonBouss(h, tv, PFu, PFv, G, GV, US, CS, ALE_CSp, p_
                 ! the pressure anomaly at the top of the layer [R L4 T-4 ~> Pa m2 s-2].
   real, dimension(SZI_(G),SZJ_(G))  :: &
     dp, &       ! The (positive) change in pressure across a layer [R L2 T-2 ~> Pa].
-    SSH, &      ! The sea surface height anomaly, in depth units [Z ~> m].
+    pbot_anom, & ! Bottom pressure (anomaly) used for self-attraction and loading. If SAL_USE_BPA
+                 ! is False, pbot_anom is sea surface height with a depth unit. Otherwise pbot_anom
+                 ! is the total bottom pressure [Z ~> m or R L2 T-2 ~> Pa].
```

Comment:
The syntax of the units here is a bit different than what we have used elsewhere  Elsewhere the ' or ' inside of the square brackets is reserved for the unit changes when non-Boussinesq.  In this case pbot_anom has one of two different units, so it would be `[Z ~> m] or [R L2 T-1 ~> Pa]`.  Also, I think that the code might be clearer (and easier to read) if it had retained the `SSH` variable for when `SAL_USE_BPA` is false, and only used `pbot_anom` when `SAL_USE_BPA` is true.  Please consider making this change to retain the two separate variables, even though this would mean two separate calls to `calc_SAL()` at about lines 416 and 424.

  The same consideration would apply to the `pbot_anom` array introduced in `PressureForce_FV_Bouss()` at line 878.